### PR TITLE
Add fetchpriority attribute for cover image on product detail page

### PR DESCRIPTION
**Pull request by jesperingels**
### 1. Why is this change necessary?
Adding the `fetchpriority="high"` attribute to the LCP (Largest Contentful Paint) will make sure the LCP loads faster and will improve the performance. This PR is specifically for the LCP on the product detail pages -> the cover image. 

### 2. What does this change do, exactly?
This change adds the attribute `fetchpriority="high"` for the first image in the product gallery. The browser will load this image faster than it currently does. For more information about Fetch Priority read [web.dev - Fetch Priority](https://web.dev/articles/fetch-priority)

### 3. Describe each step to reproduce the issue or behaviour.
When you visit a product detail page in Shopware and inspect the image element of the first image of the product gallery you'll see that the attribute `fetchpriority="high"` is currently not present. 

### 5. Checklist

- [x] I have rebased my changes to remove merge conflicts
- [x] I have written tests and verified that they fail without my change
- [x] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-implement-new-changelog.md) with all necessary information about my changes
- [ ] I have written or adjusted the documentation according to my changes
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: src/Storefront/Resources/views/storefront/utilities/thumbnail.html.twig [hunk 1]
```diff
@@ -33,6 +33,10 @@
         {% set attributes = attributes|merge({ loading: 'eager' }) %}
     {% endif %}
 
+    {% if attributes.fetchpriority == 'high' %}
+        {% set attributes = attributes|merge({ 'fetchpriority': 'high' }) %}
+    {% endif %}
```
Comment (PheysX) [inline src/Storefront/Resources/views/storefront/utilities/thumbnail.html.twig]: Do we actually need this condition?

Comment (jesperingels) [reply src/Storefront/Resources/views/storefront/utilities/thumbnail.html.twig]: Yes definitely! Without this condition every image would always get `fetchpriority="high"`, which is not something you want. Always having `fetchpriority="high"` on an image can potentially be bad for performance.

Comment (PheysX) [reply src/Storefront/Resources/views/storefront/utilities/thumbnail.html.twig]: I also meant the content inside. 

You check for the key fetchpriority = high and assign fetchpriority = high
You don't need to assign something what already there 😄 

Shouldn't it also work without changes on the thumbnail file?

Comment (jesperingels) [reply src/Storefront/Resources/views/storefront/utilities/thumbnail.html.twig]: Hmm good point! 😅 
I'll change it!

**Comment by tobiasberge** [comment]
Hi @jesperingels,

thank you for you contribution and nice improvement. 🎉 👍

**Review by tobiasberge — APPROVED** [review]

**Comment by tobiasberge** [comment]
Hi @jesperingels,

we have merged your changes to trunk: https://github.com/shopware/shopware/commit/d322c220b94c4edb38993d303717d45f484731f6

**Pull request closed without merging**